GameFramework/Color/Color.h

```diff
@@ -0,0 +1,231 @@
+﻿#ifndef COLOR_H
+#define COLOR_H
+
+#include <Windows.h>
+
+#include <d3dx9.h>
+
+/// <summary>
+/// 基礎構築に関するものをまとめた名前空間
+/// </summary>
+namespace gameframework
+{
+	/// <summary>
+	/// 色保存するための構造体
+	/// </summary>
+	struct Color
+	{
+	public:
+		/// <summary>
+		/// 色の構成要素
+		/// </summary>
+		enum class COLOR_COMPONENTS
```

Comment:
C++11は自信無いのですが、使うときは `Color::COLOR_COMPONENTS::ALPHA` みたいになるのでは？
だとしたら冗長なので
```suggestion
		enum class COMPONENTS
```
で良くないですか？

僕は他のやり方がわからなかったので、テストコードではエイリアス作りました。

Reply:
修正しました